tests/lib/helper.ts

```diff
@@ -20,76 +20,82 @@ export const hexToRGB = (color?: string) => {
   })`;
 };
 
+export interface CustomEventPayload {
+  detail: unknown;
+}
+
 export const waitForCustomEvent = async (
```

Comment:
Thanks so much for cleaning all this up! The changes look good. One ask, could we add tests for the helper functions?

Reply:
Sure, I'll give it a shot, but...

...`waitForEvent` / `expectNoEvent` are very coupled to the Playwright framework itself - they call `page.evaluate`, which seems to do something complicated to serialize the function over IPC to the browser, run in the browser context, and then serialize something back - so I'm not 100% certain of the value of adding such tests.

Such tests would likely be redundant with the test suite itself, because they would look something like...

1. Use Playwright to trigger an event on a DOM node, probably via `page.evaluate`
2. Use `waitForCustomEvent` to listen for that event

...which is more or less what every test that uses `waitForEvent` does

Reply:
Update: the value of writing the tests turned out to be "Teach Michael how Playwright works," so I'll chalk that one up as a win

Reply:
That's totally fair, it would be a bit redundant. I think the one case they may be helpful is if someone is trying to change the helper functions. The basic tests would be a good way to outline the intended behavior of the function and the tests wouldn't be coupled to a component test. 

That said totally your call if you want to add them!

Reply:
I agree, the tests are helpful documentation, and Playwright is _quirky_, so they function well as contract tests, too. I'll be adding them